packages/extensions/openapi-to-typespec/test/arm-agrifood/tsp-output/main.tsp

```diff
@@ -20,6 +20,7 @@ using Azure.Core;
 using Azure.ResourceManager;
 using TypeSpec.Versioning;
 @armProviderNamespace
+@useAuth(AadOauth2Auth<["user_impersonation"]>)
```

Comment:
it seems we could not get scope info from m4 for arm?

Reply:
not sure I follow, this seems to be the scope defined in the swagger
![image](https://github.com/Azure/autorest/assets/21109913/3fc53dbc-3884-4825-bf06-501aa801da67)